pom.xml

```diff
@@ -44,7 +44,7 @@
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <gitRepositoryName>sonar-scanner-msbuild</gitRepositoryName>
     <!-- Release: enable publication to Bintray -->
-    <artifactsToPublish>${project.groupId}:${project.artifactId}:zip</artifactsToPublish>
+    <artifactsToPublish>${project.groupId}:${project.artifactId}:${project.classifier}:zip</artifactsToPublish>
```

Comment:
I think the classifier should go after the type. This is a recent change made by @drautureau-sonarsource . See https://github.com/SonarSource/sonarlint-core/commit/41043532c92226da312ced01819904a5eac24ada

Reply:
Also don't forget to add the second artifact (comma separated)